### PR TITLE
Fix/quick start links in mqtt-cli documentation

### PR DESCRIPTION
**Pull request by wait-io**
**Motivation**
during onboarding, I noticed a lot of wrong links in the CLI docs, the info is still there but only reachable using the navigation bar. This is not ideal and may put people off.

**Changes**
This PR fixes all site internal links. Every link change was tested locally using jekyll

### CODE REVIEW

**Comment by gitseti** [comment]
Awesome. Thanks Daniel. 
This was mostly due to a recent restructuring of the gh-pages integration.

**Review by gitseti — APPROVED** [review]

**Comment by SgtSilvio** [comment]
relative links should work (jekyll-relative-links plugin), but the links are not referring to the files (missing .md extension)
@wait-io please try to use (proper) relative links as this works in IntelliJ, the GitHub markdown viewer and the jekyll rendered site.

**Comment by wait-io** [comment]
Sorry, I can't get relative links to work using jekyll in my local env and I don't know enough about the setup used here. Please disregard my PR.

**Comment by wait-io** [comment]
@SgtSilvio is this better? works for me at least.

File: docs/_docs/quick-start.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ redirect_from: /docs/quick_start.html
 ***
 ## Usage
 
-To install MQTT CLI on your system please follow the [Installation instructions](installation).
+To install MQTT CLI on your system please follow the [Installation instructions](../installation).
```
Comment (SgtSilvio) [inline docs/_docs/quick-start.md:11]:
```suggestion
To install MQTT CLI on your system please follow the [Installation instructions](installation.md).
```

File: docs/_docs/quick-start.md [hunk 2]
```diff
@@ -40,12 +40,12 @@ $ mqtt [flags] [METHOD] [OPTION [OPTION]]
...
-* [Shell](shell)
-* [Test](test) 
-* [HiveMQ](hivemq)
-* [Swarm](swarm) 
+* [Publish](../publish)
+* [Subscribe](../subscribe)
+* [Shell](../shell)
+* [Test](../test) 
+* [HiveMQ](../hivemq)
+* [Swarm](../swarm)
```
Comment (SgtSilvio) [inline docs/_docs/quick-start.md:48]:
```suggestion
* [Publish](publish.md)
* [Subscribe](subscribe.md)
* [Shell](shell.md)
* [Test](test.md) 
* [HiveMQ](hivemq.md)
* [Swarm](swarm.md) 
```

**Comment by SgtSilvio** [comment]
@wait-io I made a few examples what I mean.
Referencing the file directly is the right way, as this links the sources and so works everywhere, also on GitHub (https://github.com/hivemq/mqtt-cli/blob/master/docs/_docs/quick-start.md)

**Comment by SgtSilvio** [comment]
superseded by #299

**Pull request closed without merging**